### PR TITLE
Weather fetch

### PR DESCRIPTION
**Pull request by tennjugu**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    .then(displayWeatherInfo)
+    .catch(onError)
+})
+
+clearBtn.addEventListener('click', (event) =>{
+    event.preventDefault()
+    clearSearch()
+})
+
+form = document.querySelector('form')
```
Comment (zizoh) [inline script.js]: Appears this form variable was not declared in this file, or did I miss it?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+function errorDisplay(){
+    const errorDisplay = document.querySelector('.error')
+    if (errorDisplay){
+        errorDisplay.remove()
+    }
+}
+
+function clearSearch(){
+    form = document.querySelector('form')
```
Comment (zizoh) [inline script.js]: This was already don on line 18, right?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+function getFetchedData(response) {
+    if (!response.ok) {
+        const errorData = response.json()
+        throw new Error(errorData.error.message)
+    }
+    return response.json()
+}
+
+function displayWeatherInfo(weatherInfo) {
```
Comment (zizoh) [inline script.js]: This function is too long. Please consider splitting it. Can show the current weather and the forecast in different functions.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        element.addEventListener('click', () => {
+            element.style.display = 'none'
+            const correspondingCTemp = element.previousElementSibling
+            correspondingCTemp.style.display = 'flex'
+        })
+    })
+
+    currentTempC.addEventListener('click', () =>{
+        currentTempC.style.display = 'none'
+        const correspondingTempF = currentTempC.nextElementSibling
```
Comment (zizoh) [inline script.js]: This variable should have been named correspondingTempC and not correspondingTempF, right?

File: script.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+function resetDisplay(){
+    const weatherDisplayDiv = document.querySelector('.weatherDisplay')
+    if(weatherDisplayDiv){
+        weatherDisplayDiv.remove()
+    }
+}
+
+function toggleTemp(){
+    const cTemp = document.querySelectorAll('.c-temp')
```
Comment (zizoh) [inline script.js]: This function can be split into 4 different functions for handling the click events

Comment (tennjugu) [reply script.js]: I’ll affect these corrections

**Review by zizoh — APPROVED** [review]

**Pull request merged**